en/06_Project_Governance/06_Minor_release_policy.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Minor Release Policy
+summary: Outline the minor release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Minor Release Policy
+
+## Scope of the policy
+
+This policy applies to all [Silverstripe CMS commercially supported modules](/project_governance/supported_modules/).
+
+Community modules are not covered by this policy. Modules in the `silverstripe` github organisation that are not commercially supported are updated on a best effort basis.
+
+## General approach to minor releases
+
+This policy complements the [Silverstripe CMS Major release policy](major_release_policy) by providing guidance on when new minor releases of Silverstripe CMS are shipped.
+
+Note that the release cadence and pre-release time frame are indicative only. We may alter those in response to external constraints.
+
+Our minor release policy is more flexible because minor release upgrades are more straight forward to manage than major release upgrades. 
+
+## Upgrading to a new minor release
+
+Silverstripe CMS follows [semantic versioning](https://semver.org/). Silverstripe CMS minor releases deliver new features and new APIs in a backward compatible way.
+
+## Minor release cadence
+
+Silverstripe CMS aims to ship a new minor releases every 6 months for the major release line in active development. Those minor releases are targeted for April and October each year.
+
+### Pre-release
+
+Silverstripe CMS does not usually tag alpha releases for minor releases.
+
+Approximately 6 weeks prior to the anticipated stable minor release, a beta minor release is published. Once a beta release is tagged, any new feature or API that didn't make it to the beta should be targeted to the follow up minor release. This allows the CMS development team to perform a regression test on the beta release with confidence that no additional regressions will be introduced before the stable release.
+
+
+Approximately 2 weeks prior to the anticipated stable minor release, a release candidate is tagged. Once a release candidate is tagged, only critical impact bug fixes can be added to the release. The release candidate is sent to an external auditor for a security review. Any security patches which will be included in the stable release are also sent to the auditor.
+
+### Stable release
+
+New minor releases of Silverstripe CMS are tagged around April and October each year.
+
+## Minor release support timeline
+
+The minor release support timeline follows similar phases to those of a major release, but are more condensed.
+
+### Bug and security fixes
+
+A Silverstripe CMS minor release line enters the *bug and security fixes* phase once it is tagged *stable*.
+
+A minor release in the *bug and security fixes* phase receives:
+- regular patches for bugs at all impact levels
+- security patches for all vulnerabilities.
```

Comment:
```suggestion
- bugfixes and enhancements that do not change existing APIs
- security patches for vulnerabilities at all impact levels.
```

Reply:
Applied the suggestion mostly as-is, but removed the bit about "enhancements". I could imagine some pretty substantial enhancement that wouldn't change APIs that we would never ship in a patch.